docs/windows_dev.md

```diff
@@ -0,0 +1,98 @@
+# Integration Tests: Windows Local Setup
+**Prerequisites**: 
+- You've setup Go and the buildkit repo and you can build from source.
```

Comment:
Can add one optional pre-req since you mentioned it later:
```
- Optional: You have installed the [Delve debugger](https://github.com/go-delve/delve)
```